frontend/src/app/modules/auth/register-form/register-form.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="registration-container">
+    <img src="../../../../assets/icons/logo.svg" alt="logo" />
+    <div class="to-login-container">
+        <h2>Create an account</h2>
+        <p class="already-registered">Already registered? <span>Log in</span></p>
+    </div>
+    <form class="registration-form" [formGroup]="registerForm" action="">
+        <p class="input-label">Full Name</p>
+        <mat-form-field class="input-form" appearance="fill" color="warn" floatLabel="always" hideRequiredMarker="true">
+            <mat-label>Enter your name</mat-label>
```

Comment:
I think we can get rid of labels in here, since there are `<p class="input-label">Full Name</p>`. I like those more.
Or move labels up and remove paragraphs so the are clickable and make inputs active.